### PR TITLE
More detailed steps for configuring telegram

### PR DESCRIPTION
**Pull request by mmartinortiz**
Small changes on the instructions for the user about enabling Telegram and for closing the issue #108 

I detected that the [`GatewaySettings.vue`](https://github.com/nextcloud/twofactor_gateway/blob/9e220d31d5d6540b558fe44fee7d08ee56551493/js/components/GatewaySettings.vue#L25) file states "*Enter your identification (e.g. phone number to start the verification):*". I think the telephone number does not apply to Telegram.

### CODE REVIEW

File: js/components/TelegramInstructions.vue [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+			to setup a Telegram Bot that will send the authentication codes to you.
+			Once this has been done:<br>
+			<ol>
+			  <li>Start a conversation with the Bot. You can ask your admin for
+				the bot name.</li>
+				<li>You need your Chat Id. You can get it sending a message to the <a
+					href="https://telegram.me/get_id_bot"
+					target="_blank"
+					rel="noreferrer noopener">ID Bot</a>.</li>
+				<li>Introduce your Chat Id in the following box. You will receive a
```
Comment (ChristophWurst) [inline js/components/TelegramInstructions.vue]: use
```
<li>
    ...text....
</li>
```
formatting please :wink:

Comment (mmartinortiz) [reply js/components/TelegramInstructions.vue]: Done

File: js/components/TelegramInstructions.vue [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+			In order to receive authentication codes via Telegram, your admin needs
+			to setup a Telegram Bot that will send the authentication codes to you.
+			Once this has been done:<br>
+			<ol>
+			  <li>
+					Start a conversation with the Bot. You can ask your admin for
+					the bot name.
+				</li>
+				<li>
+					You need your Chat Id. You can get it sending a message to the <a
```
Comment (ChristophWurst) [inline js/components/TelegramInstructions.vue]: I don't think this is correct. What the user enter is their *user ID*. See #110

Comment (mmartinortiz) [reply js/components/TelegramInstructions.vue]: Do you agree to include a more detailed description of Chat Id and Group Id based on the comment [430174531](https://github.com/nextcloud/twofactor_gateway/issues/110#issuecomment-430174531)?

I can rephrase it to make it sorter

Comment (ChristophWurst) [reply js/components/TelegramInstructions.vue]: In the light of our findings in #110 I think the whole setup process should change a bit and remove the current (and buggy) setup logic to just ask the user for their user id. I will look into this soonish.

**Comment by ChristophWurst** [comment]
closing due to lack of activity
